packages/ui/src/domainModules/collectionMammals/components/MammalManager/ResultOptionsBar/index.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Button, Grid, Icon, Menu, Popup, Segment } from 'semantic-ui-react'
+import 'common/dist/semantic.css' // eslint-disable-line
+import 'react-rangeslider/lib/index.css'
+
+const propTypes = {
+  activeItem: PropTypes.string.isRequired,
+  onExportCSV: PropTypes.func.isRequired,
+  onFormTabClick: PropTypes.func.isRequired,
+  onSettingClick: PropTypes.func.isRequired,
+  onTableTabClick: PropTypes.func.isRequired,
+  tabContent: PropTypes.string.isRequired,
+}
+
+export class ResultOptionsBar extends Component {
+  render() {
+    const {
+      activeItem,
+      onExportCSV: handleExportToCSV,
+      onFormTabClick: handleFormTabClick,
+      onSettingClick: handleSettingClick,
+      onTableTabClick: handleTableTabClick,
+      tabContent,
+    } = this.props
+
+    return (
+      <div>
+        <Menu attached="top" tabular>
+          <Menu.Item
+            active={activeItem === 'form'}
+            name="form"
+            onClick={() => handleFormTabClick()}
+          >
+            <Icon name="wordpress forms" size="large" />
+          </Menu.Item>
+          <Menu.Item
+            active={activeItem === 'table'}
+            name="table"
+            onClick={() => handleTableTabClick()}
+          >
+            <Icon name="table" size="large" />
+          </Menu.Item>
+          <Menu.Menu position="right">
+            <Grid textAlign="center" verticalAlign="middle">
+              <Grid.Column>
+                <Popup
+                  content={
+                    <Button
+                      content="Export result to csv"
+                      onClick={() => handleExportToCSV()}
+                    />
+                  }
+                  on="click"
+                  trigger={<Icon name="share" size="large" />}
+                />
+              </Grid.Column>
+            </Grid>
+
+            <Menu.Item>
+              <Icon
+                name="setting"
+                onClick={() => handleSettingClick()}
+                size="large"
+              />
+            </Menu.Item>
+          </Menu.Menu>
+        </Menu>
+        <Segment attached="bottom">{tabContent}</Segment>
```

Comment:
I think this should not be part of the RecordOptionsBar, we will put the tabContent in a separate component instead. Since you use the Menu instead of Tabs, they don't need to be combined like this. Instead, the RecordOptionsBar will be in its own row and below that we will have a row for the tab content. So you can remove this segment and the wrapping div. (Also, when the wrapping div doesn't have any meaning except fulfilling the requirement to have a wrapping tag, then you can use <React.Fragment> (google it for more info)